tasks/identity.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
Style thing, but Ansible's YAML parser does not need `---` at the top of files. Since the files are `*.yml` we know they're YAML going into them. 

Instead of `---` this is a good place to explain what this file is for. Ansible can be hard to organize into source control. Hints like this get us to think about the organization like a user new to the repo.